scripts/build-android.sh

```
@@ -0,0 +1,230 @@
+        ["mdpi"]=48
+        ["hdpi"]=72
+        ["xhdpi"]=96
+        ["xxhdpi"]=144
+        ["xxxhdpi"]=192
+    )
+
+    # Temp dir only needed for cwebp tool
+    local tmp_dir=""
+    if [ "$tool" = "cwebp" ]; then
+        tmp_dir=$(mktemp -d "/tmp/ic_launcher_XXXXXX")
+        trap "rm -rf \"$tmp_dir\"" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)